tests/storage_test.go

```diff
@@ -300,218 +300,230 @@ var _ = Describe("Storage", func() {
 
-					tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
-				})
+						tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
+					})
 
-				AfterEach(func() {
-					if nodeName != "" {
-						tests.RemoveHostDiskImage(diskPath, nodeName)
-					}
+					AfterEach(func() {
+						if nodeName != "" {
+							tests.RemoveHostDiskImage(diskPath, nodeName)
+						}
+					})
 				})
-			})
 
-			Context("With 'DiskExists' type", func() {
-				diskName := "disk-" + uuid.NewRandom().String() + ".img"
-				diskPath := filepath.Join(hostDiskDir, diskName)
-				// it is mandatory to run a pod which is creating a disk image
-				// on the same node with a HostDisk VMI
-				var nodeName string
-
-				BeforeEach(func() {
-					// create a disk image before test
-					job := tests.CreateHostDiskImage(diskPath)
-					job, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(job)
-					Expect(err).ToNot(HaveOccurred())
-					getStatus := func() k8sv1.PodPhase {
-						pod, err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Get(job.Name, metav1.GetOptions{})
+				Context("With 'DiskExists' type", func() {
+					diskName := "disk-" + uuid.NewRandom().String() + ".img"
+					diskPath := filepath.Join(hostDiskDir, diskName)
+					// it is mandatory to run a pod which is creating a disk image
+					// on the same node with a HostDisk VMI
+					var nodeName string
+
+					BeforeEach(func() {
+						// create a disk image before test
+						job := tests.CreateHostDiskImage(diskPath)
+						job, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(job)
 						Expect(err).ToNot(HaveOccurred())
-						if pod.Spec.NodeName != "" && nodeName == "" {
-							nodeName = pod.Spec.NodeName
+						getStatus := func() k8sv1.PodPhase {
+							pod, err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Get(job.Name, metav1.GetOptions{})
+							Expect(err).ToNot(HaveOccurred())
+							if pod.Spec.NodeName != "" && nodeName == "" {
+								nodeName = pod.Spec.NodeName
+							}
+							return pod.Status.Phase
 						}
-						return pod.Status.Phase
-					}
-					Eventually(getStatus, 30, 1).Should(Equal(k8sv1.PodSucceeded))
-				})
-
-				It("Should use existing disk image and start", func() {
-					By("Starting VirtualMachineInstance")
-					vmi := tests.NewRandomVMIWithHostDisk(diskPath, v1.HostDiskExists, nodeName)
-					tests.RunVMIAndExpectLaunch(vmi, 30)
-
-					By("Checking if disk.img exists")
-					vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
-					output, err := tests.ExecuteCommandOnPod(
-						virtClient,
-						vmiPod,
-						vmiPod.Spec.Containers[0].Name,
-						[]string{"find", hostDiskDir, "-name", diskName},
-					)
-					Expect(strings.Contains(output, diskPath)).To(BeTrue())
+						Eventually(getStatus, 30, 1).Should(Equal(k8sv1.PodSucceeded))
+					})
+
+					It("[test_id:847]Should use existing disk image and start", func() {
```

Comment:
Test ID 847 seems to lay out a different test than what's here. Please double check. I'm not saying this test is wrong, perhaps the test_id is typo-ed?

Reply:
Good catch! I should add one more test for our kubevirtci and add one more in polarion to reflect this one.

Reply:
Test added, I asked @ILpinto to add one additional scenario for positive tests.